pandas/tests/indexes/test_any_index.py

```diff
@@ -33,6 +33,20 @@ def test_mutability(index):
         index[0] = index[0]
 
 
+def test_values_mutability(index):
+    if not len(index):
+        return
```

Comment:
moot point till tests are passing, but ideally should use pytest.skip with message.